complexify-android/src/main/java/co/infinum/complexify/Constants.kt

```diff
@@ -0,0 +1,158 @@
+package co.infinum.complexify
+
+object Constants {
```

Comment:
This object must die.
In order to make maintenance easier this should be in resources or assets, depending on sensitivity point of view. 
Then loaded in memory when library is instantiated.
Also, when removed from code, adding or removing or editing will not trigger recompiling but just packaging the library.

Reply:
How do you suggest I access those without context?